simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountAuthenticationViewBindings.kt

```diff
@@ -181,6 +190,19 @@ sealed class AccountAuthenticationViewBindings {
       }
     }
 
+    override fun setResetPasswordLabelStatus(
+      status: AccountLoginButtonStatus,
+      isVisible: Boolean,
+      onClick: () -> Unit
+    ) {
+      this.resetPasswordLabel.setOnClickListener {
+        onClick()
+      }
+      this.resetPasswordLabel.isVisible = isVisible && (
+        status is AsLoginButtonEnabled || status is AsLoginButtonDisabled
```

Comment:
Why does the visibility of the password reset link depend on the status of the login button?

Reply:
Well, the feature says that the "forgot your password" label should appear when the user is not logged in, and those two status refer to that scenario. I assumed that as long as the login button is there, regardless of its state, the label should also be there (if there's an URI too). The remaining status refer to the "Cancel" and "Logout" options, so I thought the label shouldn't be visible in these cases

Reply:
Thanks, that makes sense. Those states are confusing.